web/src/server_events_dispatch.js

```diff
@@ -205,12 +205,12 @@ export function dispatch_normal_event(event) {
 
         case "realm": {
             const realm_settings = {
-                add_custom_emoji_policy: settings_emoji.update_custom_emoji_ui,
                 allow_edit_history: noop,
                 allow_message_editing: noop,
                 edit_topic_policy: noop,
                 avatar_changes_disabled: settings_account.update_avatar_change_display,
                 bot_creation_policy: settings_bots.update_bot_permissions_ui,
+                can_add_custom_emoji_group: settings_emoji.update_custom_emoji_ui,
```

Comment:
Live update is not working as expected, probably because event for group settings are wit `op=update_dict` and not `op=update`.

Reply:
Moved `settings_emoji.update_custom_emoji_ui` to `update_dict` case.